<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThemCongVanDialog</class>
 <widget class="QDialog" name="ThemCongVanDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thêm công văn</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="nguoiKyLabel">
     <property name="text">
      <string>Họ tên người kí:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="nguoiKyComboBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="soHieuLabel">
     <property name="text">
      <string>Số hiệu văn bản:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="soHieuLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="ngayBanHanhLabel">
     <property name="text">
      <string>Ngày ban hành:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDateEdit" name="ngayBanHanhDateEdit">
     <property name="displayFormat">
      <string>dd/MM/yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="khoTuLieuLabel">
     <property name="text">
      <string>Kho tư liệu:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="khoTuLieuComboBox"/>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="icon">
        <iconset resource="../MainWindow.qrc">
         <normaloff>:/icons/apply.svg</normaloff>:/icons/apply.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Hủy</string>
       </property>
       <property name="icon">
        <iconset resource="../MainWindow.qrc">
         <normaloff>:/icons/cancel.svg</normaloff>:/icons/cancel.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
